tests/test_transpile.py

```diff
@@ -21,6 +22,26 @@ class TestTranspile(unittest.TestCase):
     def validate(self, sql, target, **kwargs):
         self.assertEqual(transpile(sql, **kwargs)[0], target)
 
+    def test_functions(self):
```

Comment:
Move these to `test_clickhouse.py`, get rid of dialect imports and use string versions.

Reply:
But this test for transpiling. How to implement same test in there? Is it okay if I'll just use string-like dialect names?